incubator/kube-aws-ingress-controller/templates/daemonset.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
```

Comment:
`apps/vbeta2`

Reply:
I updated it but I don't get where did you saw that version? Looks like undocumented to me https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#ingresslist-v1beta1-extensions

Reply:
Anyway I updated it with the last commit, in case.